### PR TITLE
Update info.xml

### PR DESCRIPTION
**Pull request by rakekniven**
Fixed meaning.

Reported at Transifex.

Signed-off-by: Mark Ziegler <mark.ziegler@rakekniven.de>

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	<id>firstrunwizard</id>
 	<name>First run wizard</name>
 	<summary>A welcome and introduction wizard for Nextcloud</summary>
-	<description>This application enables a pop-up window when a user first logs into Nextcloud. This window welcomes new users to Nextcloud, and contains links to the standard desktop client, Android app, and the iOS app. The window also shows a link for accessing Nextcloud via WebDAV. At any time, the first run wizard button can be opened again via the link on the personal page.
+	<description>This application enables a pop-up window when a user first logs into Nextcloud. This window welcomes new users to Nextcloud, and contains links to the standard desktop client, Android app, and the iOS app. The window also shows a link for accessing Nextcloud via WebDAV. At any time, the first run wizard can be opened again via the link on the personal page.
```
Comment (juliushaertl) [inline appinfo/info.xml]: We moved the button to the settings menu (About) 
image

Mind to adjust this as well @rakekniven ?

Comment (rakekniven) [reply appinfo/info.xml]: Finally done.

**Pull request merged**